agent/grpc-internal/client.go

```diff
@@ -148,7 +148,9 @@ func (c *ClientConnPool) dial(datacenter string, serverType string) (*grpc.Clien
 		grpc.WithKeepaliveParams(keepalive.ClientParameters{
 			Time:    30 * time.Second,
 			Timeout: 10 * time.Second,
-		}))
+		}),
+		grpc.WithDefaultCallOptions(grpc.MaxCallSendMsgSize(50*1024*1024), grpc.MaxCallRecvMsgSize(50*1024*1024)),
```

Comment:
This gRPC client isn't used for peerstream traffic from what I remember, so I'm not sure if we need to increase its value. I suppose it would only ever prevent an issue from occurring, so it's probably safe.

Reply:
Yeah, I  noticed this error message after I fixed the peering gRPC client. I think this is from the health rpcclient to query for the health of services in the cluster.

Reply:
I think we might need this; otherwise the client agent in the peering client can't get the status of the very large services imported from peered cluster.

Reply:
Ah, okay. That makes sense.